zebra/zebra_mpls_vty.c

```diff
@@ -271,6 +271,7 @@ static int zebra_mpls_config(struct vty *vty)
 
 	write += zebra_mpls_write_lsp_config(vty, zvrf);
 	write += zebra_mpls_write_fec_config(vty, zvrf);
+	write += zebra_mpls_write_nexthop_resolution_config(vty, zvrf);
```

Comment:
This does not write anything in show run when doing
```
vrf NAME
 mpls fec nexthop-resolution
```

You have to migrate that to mgmtd

Reply:
New changes were introduced that outdates this question. 
Now the show running-config output is like this:
```
!
vrf vrf10                          
 mpls fec nexthop-resolution
exit-vrf                                  
!
````

Reply:
OK now